Client.Core/Flux/Internal/FluxCsvParser.cs

```diff
@@ -72,119 +76,154 @@ public void ParseFluxResponse(Stream source, ICancellable cancellable, IFluxResp
         {
             Arguments.CheckNotNull(source, "source");
 
-            var parsingState = ParsingState.Normal;
+            using var csv = new CsvReader(new StreamReader(source), CultureInfo.InvariantCulture);
+            ParseFluxResponseState state = new ParseFluxResponseState { csv = csv };
```

Comment:
Use 'var' (simple types):

```suggestion
            var state = new ParseFluxResponseState { csv = csv };
```